include/pcmki/pcmki_sched_allocate.h

```diff
@@ -126,7 +126,7 @@ extern gboolean clone_create_probe(pe_resource_t * rsc, pe_node_t * node, pe_act
                                    gboolean force, pe_working_set_t * data_set);
 extern void clone_append_meta(pe_resource_t * rsc, xmlNode * xml);
 
-void apply_master_prefs(pe_resource_t *rsc);
+void pcmk__apply_promotion_prefs(pe_resource_t *rsc);
```

Comment:
I just merged a giant PR supporting the OCF 1.1 role names, this is now pcmk__add_promotion_scores(). You'll need to rebase and resolve conflicts.

Reply:
Ack, saw that. I'll take care of it next time I push :)